library/misc/mapillary/README.md

```diff
@@ -0,0 +1,17 @@
+# Mappilary
```

Comment:
"Mapillary", please replace in all files, token etc
Many occurrences in this file